### ISSUE TITLE
When using adding list/collection remember last settings (paths)

### ISSUE THREAD

**Issue by TRaSH-** [Radarr/Radarr#5005]
**Is your feature request related to a problem? Please describe.**
When adding a list/collection the path always reverts back to a default path,
it would be nice if it would remember the last used path like you also have with adding a new movie.

**Describe the solution you'd like**
It would be nice if it would remember the last used path like you also have with adding a new movie.

**Describe alternatives you've considered**
None

**Additional context**

**Comment by TRaSH-** [comment]
It's still a issue (remove the stale) ;)

**Comment by xeviff** [comment]
Yes, in my case I configured the first root path as "child films" and every time I add a collection it sets by defaul the child films path and I have to change it every time on the dropdown list... (a little bit anoying...). So, the possibility of set or change the default path will work, but I think it's even better if it remembers the last option. Thanks for your time.
